.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1399,12 +1410,49 @@ public class ChatOutputAudioReference : IJsonModel<ChatOutputAudioReference>, IP
         public static bool operator !=(ChatOutputAudioVoice left, ChatOutputAudioVoice right);
         public override readonly string ToString();
     }
+    public class ChatOutputPrediction : IJsonModel<ChatOutputPrediction>, IPersistableModel<ChatOutputPrediction> {
+        public ChatOutputPredictionKind Kind { get; }
```

Comment:
It just dawned on me that since `ChatOutputPrediction` is an input only class, we don't really need the `Kind` property (just like we don't need the `Content` property either). Taking `ChatResponseFormat` as the pre-existing pattern, that class has no properties; it only has factory methods. I think we should stick to that pattern for now (and we can always add the `Kind` and/or `Content` properties in the future if needed). This also means we can remove the `ChatOutputPredictionKind` enum from the public API surface.